tmail-backend/jmap/extensions-es/src/main/java/com/linagora/tmail/james/jmap/EmailAddressContactMappingFactory.java

```diff
@@ -0,0 +1,54 @@
+package com.linagora.tmail.james.jmap;
+
+import static org.elasticsearch.common.xcontent.XContentFactory.jsonBuilder;
+
+import java.io.IOException;
+
+import org.apache.james.backends.es.v7.IndexName;
+import org.apache.james.backends.es.v7.WriteAliasName;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+public class EmailAddressContactMappingFactory {
+    // TODO configuration POJO for these index/alias settings
+    public static final IndexName INDEX_NAME = new IndexName("email_contact");
+    public static final WriteAliasName ALIAS_NAME = new WriteAliasName("email_contact_alias");
+
+    public static XContentBuilder generateSetting() throws IOException {
+        return jsonBuilder()
+            .startObject()
+                .startObject("properties")
+                    .startObject("accountId")
+                        .field("type", "text")
+                    .endObject()
+                    .startObject("id")
+                        .field("type", "text")
+                    .endObject()
+                    .startObject("address")
+                        .field("type", "text")
+                    .endObject()
+                .endObject()
+            /*
+                // TODO add the analisis / tokenizer parameter as parameters of IndexCreationFactory...
```

Comment:
Is that supposed to follow an update on James submodule?

Reply:
No it requires a ticket and a contribution in /backends-common/es-v7

Reply:
yeah that's what I had in mind with my question